color.cpp

```diff
@@ -173,7 +180,7 @@ void test_color()
 #endif // FIX_ISSUE_229
 
 
-  //#define FIX_ISSUE_230
+#define FIX_ISSUE_230

```

Comment:
This line can be romoved along with the other #ifstatements. @richelbilderbeek will make this cleaning up a newbie issue. 👍 :